package.json

```diff
@@ -41,7 +41,7 @@
   "scripts": {
     "pretest": "eslint ./ && jscs ./",
     "lint": "eslint ./ && jscs ./",
-    "test": "tap --timeout=1000 ./test/test-*"
+    "test": "tap --timeout=1000 --save=failed.txt ./test/test-*"
```

Comment:
At first I was concerned about this, but CI clears the workspace before building.. so maybe this will actually just speed up the re-test-with-debug phase..